Swiftfin tvOS/Components/LiveTVChannelItemElement.swift

```diff
@@ -177,3 +176,19 @@ struct LiveTVChannelItemElement: View {
         }
     }
 }
+
+struct LiveTVChannelItemElement_Preview: PreviewProvider {
```

Comment:
I am trying to not do previews since they can just become a pain to maintain in the long run and are also funky to use when some views have them and some don't. Even though I have left some during my development in `tvOS/AboutViewCard` and `tvOS/MainOverlay`, they can be deleted as well.

Reply:
Ok! No problem. Preview removed in latest commit.